libs/documentation/src/lib/components/formly-search/demos/advanced-search/advanced-search.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="advanced-search-container">
+  <a class ="float-right usa-link" href={{this.advancedSearchUrl}}><strong>{{this.model.advanceSearch}}</strong></a> 
```

Comment:
This does not add the option to add "Advanced Search" link to any search field. Devs would still have to add this code to get the search. Also, if there is any field specified in `fields` before the search configuration this link and the search field will become separated.

Reply:
Best to update either libs/packages/sam-formly/src/lib/formly/types/search.ts or libs/packages/components/src/lib/search/search.component.html with this link so that it can be added to any search field. Update the first to only apply the change to formly-search fields. Update the second to allow for search fields on any search field.

Reply:
Putting this in those templates would also allow the link text & url to be pulled from templateOptions rather than an additional variable in the component.